infra/main.bicep

```diff
@@ -393,7 +392,7 @@ var appEnvVariables = {
   AZURE_AUTH_TENANT_ID: tenantIdForAuth
   AZURE_AUTHENTICATION_ISSUER_URI: authenticationIssuerUri
   // CORS support, for frontends on other hosts
-  ALLOWED_ORIGIN: allowedOriginsEnv
+  ALLOWED_ORIGIN: join(allowedOrigins, ';')
```

Comment:
I removed the variable since this is only used in one place. Generally seems good to avoid excess variables since they might get misused, especially this one that overlaps so closely with other one.